lblod/ember-rdfa-editor-lblod-plugins
# GN-4262: addition of WGS84 coordinates to address variables

### Overview
This PR adds WGS84 (https://en.wikipedia.org/wiki/World_Geodetic_System) coordinates to address variables. The coordinates are fetched alongside the addresses from the location register (https://geo.api.vlaanderen.be/geolocation/v4/Location). 
The geo namespace is used (https://www.w3.org/2003/01/geo/) in order to represent the coordinates.

##### connected issues and PRs:
Jira: https://binnenland.atlassian.net/browse/GN-4262?atlOrigin=eyJpIjoiMWIxMDJiZmFhOWViNGQ1M2ExN2UzODM1N2NlMTU5MzkiLCJwIjoiaiJ9


### How to test/reproduce
- Insert and fill in an address
- Check if the coordinates are stored in the address node attributes
- Check if the coordinates are correctly serialized to HTML
- Check if the coordinates are correctly parsed from HTML
- Check if the coordinates correspond to the selected address

### Challenges/uncertainties
Aside from WGS84 coordinates, the location register also provides Lambert72 coordinates (the system employed by the Belgian geographic institute) https://nl.wikipedia.org/wiki/Lambertco%C3%B6rdinaten. Unsure what the best coordinate system would be for us to use. The WGS system seems more universal



### Checks PR readiness
